projects/dsp-ui/src/lib/action/action.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { KeyPipe } from './pipes/key.pipe';
+import { ReversePipe } from './pipes/reverse.pipe';
+import { SortByPipe } from './pipes/sort-by.pipe';
+import { StringifyStringLiteralPipe } from './pipes/stringify-string-literal.pipe';
+import { TruncatePipe } from './pipes/truncate.pipe';
+
+
+
+@NgModule({
+  declarations: [KeyPipe, ReversePipe, SortByPipe, StringifyStringLiteralPipe, TruncatePipe],
+  imports: [
+    CommonModule
+  ]
+})
+export class ActionModule { }
```

Comment:
@mdelez Careful here, the name must start with Dsp, e.g. DspActionModule

Reply:
good catch! Fixed in 7341837 :)